src/features/core/presenters/horaires/horaires.presenter.ts

```diff
@@ -153,3 +154,17 @@ export const isOpenOn =
       return false;
     }
   };
+
+export const getIntervalWeekByOffset = (weekOffset: string | number): string => {
+  const today = new Date();
```

Comment:
```suggestion
export const getIntervalWeekByOffset = (weekOffset: string | number, today: Date): string => {
const startOfTheWeek = new Date(today);
```
Cette fonction serait plus facile à tester sans créer une date qui pourrait varier au cours du temps, il faut mieux la passer en paramètre